### PR TITLE
Add new speakers and presentations for Meeting #12

### PR DESCRIPTION
**Pull request by kroucis**

### CODE REVIEW

File: data.rkt [hunk 1]
```diff
@@ -274,6 +278,33 @@
...
+                  #:email ""
+                  #:website #f
+                  #:affiliated-links '())
+
+               (S #:id speaker-id-daniel-krol
+                  #:name "Daniel Krol"
+                  #:email ""
+                  #:email-show? #f
+                  #:website #f
+                  #:affiliated-links (list (u "https://danielkrol.com/")))
```
Comment (xandkar) [inline data.rkt]: Personal website should go into `#:website`.

File: data.rkt [hunk 2]
```diff
@@ -114,9 +114,13 @@
...
 (define speaker-id-jared                    "jared")
+(define speaker-id-tigran-khandkar          "tigran-khandkar")
+(define speaker-id-jack-jutzi               "jack-jutzi")
+(define speaker-id-lisa-???                 "lisa-???")
+(define speaker-id-daniel-krol              "daniel-krol")
 
-(define host-id-raven-labs            "raven-labs")
-(define host-id-manchester-makerspace "manchester-makerspace")
+(define host-id-raven-labs                  "raven-labs")
+(define host-id-manchester-makerspace       "manchester-makerspace")
```
Comment (xandkar) [inline data.rkt]: This is unrelated to the PR - these IDs did not change.

File: data.rkt [hunk 3]
```diff
@@ -753,7 +784,49 @@
...
+              #:description "A learning email newsletter and web service written from scratch in the Rust programming language."
+              #:artifacts (list (Link #f (u "https://github.com/jsjutzi/rust-zero-backend")))
+              #:website #f
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-daniel-krol)
+              #:title "Self-hosted Navigation Maps."
+              #:description "Using Sandstorm and other distributed technologies to host your own Google Maps replacement with social features and more!"
+              #:artifacts '()
+              #:website #f
```
Comment (xandkar) [inline data.rkt:828]: @orblivion what should the website for this project be?

Comment (xandkar) [reply data.rkt:828]: Also, any links to source artifacts?

File: data.rkt [hunk 4]
```diff
@@ -753,7 +784,49 @@
...
+              #:references '())
+
+           (T #:speaker (speaker speaker-id-lisa-???)
+              #:title "PiHole and self-hosted DNS"
+              #:description "Using PiHole to control DNS resolution, monitoring ads and tracking requests from apps as well as potentially blacklisting sites and providing fine-grained parental controls."
+              #:artifacts '()
+              #:website #f
+              #:references '())
```
Comment (xandkar) [inline data.rkt]: I'd add https://pi-hole.net/ to references.

File: data.rkt [hunk 5]
```diff
@@ -753,7 +784,49 @@
...
+
+           (T #:speaker (speaker speaker-id-kevin-kadow)
+              #:title "Manchester Makerspace's Custom Doorbell"
+              #:description "A networked and remote doorbell and surveillance system for the Manchester Makerspace."
+              #:artifacts '()
+              #:website #f
+              #:references '())
```
Comment (xandkar) [inline data.rkt]: Am I remembering correctly that this was built with ESP32? This maybe a good ref to add: https://en.wikipedia.org/wiki/ESP32

Hoping Kevin will chime in with more!

File: data.rkt (hunk 2 above)
Comment (kroucis) [reply data.rkt]: Should this spacing fix be removed, then?

Comment (xandkar) [reply data.rkt]: Yes

File: data.rkt (hunk 3 above)
Comment (orblivion) [reply data.rkt:828]: By "source artifacts" do you mean "releases" on Github, or similar? I have nothing like that. But here's the source:

https://github.com/orblivion/sandstorm-share-a-map

It's also the best choice for the project's website, as it links to the app on the Sandstorm app market (still as an "experimental" app).

Comment (xandkar) [reply data.rkt:828]: > By "source artifacts" do you mean "releases" on Github, or similar? I have nothing like that. But here's the source:
> 
> https://github.com/orblivion/sandstorm-share-a-map
> 

I just meant the source itself being an artifact related the presentation, so yes - this is great. Some other things could slides or whatever supporting materials. Anything you made is an artifact, in that sense.

> It's also the best choice for the project's website, as it links to the app on the Sandstorm app market (still as an "experimental" app).

:+1:
